cpp/src/Ice/IPEndpointI.cpp

```diff
@@ -290,6 +290,7 @@ vector<ConnectorPtr>
 IceInternal::IPEndpointI::connectors(const vector<Address>& addresses, const NetworkProxyPtr& proxy) const
 {
     vector<ConnectorPtr> connectors;
+    connectors.reserve(addresses.size());
```

Comment:
vector has a ctor that takes a size_t, but it's not a "reserve" constructor (unlike C#). It adds n default elements.